docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/05 - Visits.md

```diff
@@ -0,0 +1,44 @@
+# Visits
+
+## Overview
+The **Visit** designates the instance in time where a candidate has visited or contacted a site and underwent any sort of data collection.
+
+**Visit** and **Timepoint** are terms used interchangeably in LORIS. **Visits** are defined in the `Visit_Windows` table of the database.
+
+## Adding Options
+
+### Front End
+ _not yet available_
+
+### SQL (Recommended)
+
+This table must be populated with visit labels - the Imaging Pipeline critically depends on this.  To populate with visit labels, run tools/populate_visit_windows.php - or manually insert study-specific information:
```

Comment:
```suggestion
This table must be populated with visit labels - the Imaging Pipeline critically depends on this.  To populate the table with visit labels, run tools/populate_visit_windows.php - or manually insert study-specific information:
```

Reply:
@ridz1208  Still need to add 'the table' in the last sentence.

Reply:
sorry, should be done now